### PR TITLE
task/setup

### PR DESCRIPTION
**Pull request by Testablekarma**

### CODE REVIEW

**Review by chrisob55 — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -22,4 +23,4 @@ npm-debug.log
 yarn-debug.log
 yarn-error.log
```
Comment (paulb00th) [inline .gitignore:24]: Do we need all this yarn and stuff in here?

File: .swiftpm/xcode/xcuserdata/markwebb.xcuserdatad/xcschemes/xcschememanagement.plist [hunk 2]
```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>SchemeUserState</key>
+	<dict>
+		<key>DeviceKitPlayground (Playground) 1.xcscheme</key>
```
Comment (paulb00th) [inline .swiftpm/xcode/xcuserdata/markwebb.xcuserdatad/xcschemes/xcschememanagement.plist:7]: What are the various playgrounds for?

**Review by paulb00th — COMMENTED** [review]
Some questions for you

File: .swiftpm/xcode/xcuserdata/markwebb.xcuserdatad/xcschemes/xcschememanagement.plist (hunk 2 above)
Comment (Testablekarma) [reply .swiftpm/xcode/xcuserdata/markwebb.xcuserdatad/xcschemes/xcschememanagement.plist:7]: This should probz be .gitignored as it's xcuserdata! Good spot :D

File: .gitignore (hunk 1 above)
Comment (Testablekarma) [reply .gitignore:24]: These are autocreated I'm happy to remove if you want?

**Review by paulb00th — APPROVED** [review]

**Pull request merged**